File: quilt/src/main/java/org/bread_experts_group/bacillus/quilt/BacillusQuilt.java
package org.bread_experts_group.bacillus.quilt;

import org.bread_experts_group.bacillus.Bacillus;
import org.quiltmc.loader.api.ModContainer;
import org.quiltmc.qsl.base.api.entrypoint.ModInitializer;

public final class BacillusQuilt implements ModInitializer {
    @Override
    public void onInitialize(ModContainer modContainer) {
        Bacillus.init();
    }
}

File: quilt/src/main/kotlin/org/bread_experts_group/bacillus/quilt/BacillusQuilt.kt
package org.bread_experts_group.bacillus.quilt

import org.bread_experts_group.bacillus.Bacillus.init
import org.quiltmc.loader.api.ModContainer
import org.quiltmc.qsl.base.api.entrypoint.ModInitializer

class BacillusQuilt : ModInitializer {
    override fun onInitialize(modContainer: ModContainer) {
        init()
    }
}
